plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -68,6 +68,13 @@ const config = {
 		{
 			name: 'Chrome',
 			use: { ...devices[ 'Desktop Chrome' ] },
+			testDir: 'tests',
+			testIgnore: '**/products/block-editor/**', // Ignore 'block-editor' tests.
+		},
+		{
+			name: 'chrome-new-editor',
```

Comment:
`new-editor` is rather generic, maybe change this to `chrome-new-product-editor-tests`?

Reply:
Good idea! I used the name you suggested (`chrome-new-product-editor`), but removed the last `tests`.